src/containers/ProfilePage/ProfilePage.duck.js

```diff
@@ -136,14 +140,21 @@ export const queryUserListings = (userId, config) => (dispatch, getState, sdk) =
   } = config.layout.listingImage;
   const aspectRatio = aspectHeight / aspectWidth;
 
-  return sdk.listings
-    .query({
-      author_id: userId,
-      include: ['author', 'images'],
-      'fields.image': [`variants.${variantPrefix}`, `variants.${variantPrefix}-2x`],
-      ...createImageVariantConfig(`${variantPrefix}`, 400, aspectRatio),
-      ...createImageVariantConfig(`${variantPrefix}-2x`, 800, aspectRatio),
-    })
+  const queryParams = {
+    include: ['author', 'images'],
+    'fields.image': [`variants.${variantPrefix}`, `variants.${variantPrefix}-2x`],
+    ...createImageVariantConfig(`${variantPrefix}`, 400, aspectRatio),
+    ...createImageVariantConfig(`${variantPrefix}-2x`, 800, aspectRatio),
+  };
+
+  const listingsPromise = ownProfileOnly
+    ? sdk.ownListings.query(queryParams)
```

Comment:
Add a filter. 

This now includes draft & pending-approval & closed listings too.
![Screenshot 2024-10-01 at 13 54 35](https://github.com/user-attachments/assets/62b73b1d-1170-44c8-86fd-95e0a9627263)

Reply:
Fixed to only include published listings 👍